mpmp/utilities/plot_utilities.py

```diff
@@ -264,34 +281,158 @@ def plot_heatmap(heatmap_df, results_df):
     cbar.outline.set_edgecolor('black')
     cbar.outline.set_linewidth(1)
 
+    ax = plt.gca()
```

Comment:
I love how matplotlib (or I guess seaborn in this case) code reads like C code. Who needs full word variable names anyway hahaha